### PR TITLE
feat: adding baseline of fincnce terminal

### PR DESCRIPTION
**Pull request by darkfadr**
Adding the base for the Apollo Finance Terminal

### CODE REVIEW

File: frontend/app/crypto/[ticker]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    query: GET_TICKER_DETAILS,
+    variables: { ticker: decodeURIComponent(ticker) },
+    errorPolicy: "ignore",
+  })
+
+  if (error) {
+    console.error(error)
+  }
+
+  console.log(data)
```
Comment (kaushiksahoo2000) [inline frontend/app/crypto/[ticker]/page.tsx]: do we need this console.log here?

File: frontend/app/stocks/[ticker]/components/News.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                  text={article.title}
+                  url={article.link}
+                  image={article.thumbnail?.resolutions[0]?.url}
+                />
+              ))}
+          </div>
+          <ScrollBar orientation="horizontal" />
+        </ScrollArea>
+      </div>
+      {/* <div className="w-4/5">
```
Comment (kaushiksahoo2000) [inline frontend/app/stocks/[ticker]/components/News.tsx]: this commented out block needed?

File: frontend/components/atricle-card.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  publishedDate: string
+  site: string
+  text?: string
+  image?: string
+  url: string
+}
+
+export function Article(
+  { title, publishedDate, site, text, image, url }: ArticleCardProps = {
+    title: "Abu Dhabi to Host Inaugural Middle East Blockchain Awards",
```
Comment (kaushiksahoo2000) [inline frontend/components/atricle-card.tsx]: do we need this hardcoded stuff?

File: frontend/components/components-stock-performance-chart.tsx [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+  ChartContainer,
+  ChartTooltip,
+  ChartTooltipContent,
+} from "@/components/ui/chart"
+import { Button } from "@/components/ui/button"
+import { useSuspenseQuery } from "@apollo/client"
+import { GET_TICKER_DATA } from "@/lib/graphql/queries"
+import { GET_TICKER_MOCK } from "@/lib/mocks"
+
+// Mock data for different time periods
```
Comment (kaushiksahoo2000) [inline frontend/components/components-stock-performance-chart.tsx:33]: do we still need this?

File: frontend/components/components-stock-performance-chart.tsx [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+const timePeriods = ["1D", "1W", "1M", "3M", "1Y", "5Y"] as const
+type TimePeriod = (typeof timePeriods)[number]
+
+type StockPerformanceChartProps = {
+  symbol: string
+  type: "stock" | "crypto"
+}
+
+// const CustomTooltip = ({ active, payload, label }: any) => {
```
Comment (kaushiksahoo2000) [inline frontend/components/components-stock-performance-chart.tsx:107]: do we still need this?

**Review by kaushiksahoo2000 — APPROVED** [review]

File: frontend/components/atricle-card.tsx (hunk 3 above)
Comment (darkfadr) [reply frontend/components/atricle-card.tsx]: nah can clean up

**Pull request merged**